packages/forms/resources/views/components/tags-input.blade.php

```diff
@@ -24,11 +24,14 @@
         }}
     >
         <div x-show="state.length || @js(! $isDisabled)">
-            <div @class([
-                'block w-full transition duration-75 divide-y rounded-lg shadow-sm sm:text-sm border overflow-hidden focus-within:ring-1 dark:divide-gray-600',
-                'border-gray-300 focus-within:border-primary-500 focus-within:ring-primary-500 dark:border-gray-600 dark:focus-within:border-primary-500' => ! $errors->has($statePath),
-                'border-danger-600 ring-danger-600 dark:border-danger-400 dark:ring-danger-400' => $errors->has($statePath),
-            ])>
+            <div 
+                x-bind:class="{ 'divide-y dark:divide-gray-600': state?.length }"
```

Comment:
Could you try adding a top border to the tags div instead?
Would require no JS.

Reply:
Done!